compiler/rustc_expand/src/expand.rs

```diff
@@ -1319,10 +1319,12 @@ impl InvocationCollectorNode for P<ast::Item> {
 
             let mut idents = Vec::new();
             collect_use_tree_leaves(ut, &mut idents);
-            return idents;
+            idents
+        } else if let Some(ident) = self.kind.ident() {
```

Comment:
```suggestion
        } else { self.kind.ident().into_iter().collect() }
```
I think this would also work.